docs/API/helpers/dynamic-components.md

```
@@ -17,6 +17,160 @@ When the above property is set, the `parent` property of your instance will refe
+Fliplet.DynamicContainer.getAll({ id: 123 }).then(containers => {
+  console.log(containers);
+});
+```
+
+---
+
+### Container instance API (returned by `.get()` / `.getAll()`)
+
+A **Container** instance exposes the properties and methods below.
+
+- **id** `Number` — Instance id.
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 0

(MD007, ul-indent)